invenio_pidstore/version.py

```diff
@@ -30,4 +30,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "1.0.0a10.dev20160704"
+__version__ = "1.0.0b1.dev20160817"
```

Comment:
in reply to https://github.com/inveniosoftware/invenio-pidstore/pull/82#discussion_r75079764